poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
```

Comment:
I can't seem to downgrade my poetry version from 2.0.1 to 1.8.4. I wonder if this matters since we are going to upgrade to poetry 2.x eventually.

Reply:
Might have to uninstall and reinstall, or use a tool like `pipx` to have them coexist for now. It *looks* like this file is considered acceptable under Poetry 1.x even with these additions but I'd prefer to avoid making that change here unnecessarily.